presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -213,7 +219,8 @@ public void printFinalInfo()
     private void printQueryInfo(QueryResults results)
     {
         StatementStats stats = results.getStats();
-        Duration wallTime = nanosSince(start);
+        long now = System.nanoTime();
+        Duration wallTime = succinctNanos(now - start);
```

Comment:
My thought is the timestamp (that moment) we use to calculate the wallTime and ElapsedTime, should be the same. If calling `nanoSince()` twice will calculate both with different ending time (as System.nanoTime() get called twice), although, it might be slightly different.